### ISSUE TITLE
Create Separate HAL links for Archive Sets and Collection Sets

### ISSUE THREAD

**Issue by clarkepeterf** [project-lux/lux-middletier#87]
**Problem Description:**
#77 updated the `setItemsWIthImages` HAL link to check the classification of the Set in the search criteria. However, this feels redundant to the User as they know the set they were looking at is an archive. The middle tier logic should instead check if the set is an archive before building the HAL link
**Expected Behavior/Solution:**
The middle tier logic should instead check if the set is an archive before building the HAL link

**Requirements:**

**Needed for promotion:**

~~- [ ] Wireframe/Mockup - Mike~~
~~- [ ] Committee discussions - Sarah~~
~~- [ ] Feasibility/Team discussion - Sarah~~
~~- [ ] Backend requirements - TBD~~
~~- [ ] Frontend requirements- TBD~~
~~- [ ] Are new regression tests required for QA - Amy~~
~~- [ ] Questions~~
~~- List of questions for discussions. Answers should be documented within the issue.~~

**UAT/LUX Examples:**

- The "View records from this archive with images" link on [this page](https://lux-front-tst.collections.yale.edu/view/set/16ec4dbd-f7c0-4599-8b23-36f4394f9bb7) should not have any "Categorized As" fields in Advanced Search. It should look like this:

<img width="645" alt="Screenshot 2024-07-31 at 11 47 17 AM" src="https://github.com/user-attachments/assets/96e53138-8e49-4625-8176-5f865ed0cc8e">

**Dependencies/Blocks:**

~~- Blocked By: Issues that are blocking the completion of the current issue.~~
~~- Blocking: Issues being blocked by the completion of the current issue.~~

**Related Github Issues:**

- #77 

**Related links:**

- https://www.bugherd.com/projects/284041/tasks/2435

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by clarkepeterf** [comment]
This is not part of the "Sets" work that we are doing right now.

It is related to sets in a different way.

**Comment by roamye** [comment]
Looks good in PRD. Closing.
